include/robot_interfaces/robot_backend.hpp

```diff
@@ -42,6 +42,27 @@ template <typename Action, typename Observation>
 class RobotBackend
 {
 public:
+    enum TerminationReason : int
+    {
+        // ## non-failure cases (use positive numbers here)
+
+        //! Backend is still running.
+        RUNNING = 0,
```

Comment:
optional detail : I would call this rather "NOT_TERMINATED"

Reply:
done